src/core/server/logging/logging_config.ts

```diff
@@ -153,7 +168,26 @@ export class LoggingConfig {
       } as AppenderConfigType,
     ],
     [
+      // By default, we transparently rewrite `LogMeta` sent to the
+      // console appender to remove potentially sensitive keys.
+      // This logic is also hardcoded in the http response logger
+      // itself, but included here as an additional measure for
+      // other loggers which might be logging the same data.
       'console',
+      {
+        type: 'rewrite',
+        appenders: ['rewrite-console'],
+        policy: {
+          type: 'meta',
+          mode: 'update',
+          properties: SENSITIVE_META_PATHS.map((path) => ({ path, value: REDACTED_META_TEXT })),
```

Comment:
Can be removed now since you filter the sensitive headers during `LogRecord` creation?

Reply:
It depends on whether we want to try to filter any other ECS log meta by default or not. But TBH I don't know who else would be logging request/response headers in this way, so it's probably ok to remove.

Reply:
> I don't know who else would be logging request/response headers

Even if another plugin logged them, it would be logged under a different context name, so the filtering mechanism doesn't filter them out.

Reply:
> it would be logged under a different context name

Correct, I meant that the defaults applied in `logging_config` would apply to any other context name, unlike the hardcoded filter in the http response logs.

However as discussed in another thread here, this only would work until someone is no longer using the default logging config. So the cost of keeping this extra code in place may not be worth the benefit of filtering the headers in this very specific scenario (another plugin logging headers in ECS format when Kibana has a default logging config)

Reply:
I went ahead and removed this for now. Perhaps best to wait and revisit if we decide to pursue https://github.com/elastic/kibana/issues/92082